value.h

```diff
@@ -164,16 +165,8 @@ static inline void truncval_to_flags(value_t *dst, match_flags flags)
 {
     assert(dst != NULL);
 
-    dst->flags.u64b &= flags.u64b;
```

Comment:
`truncval()` should always be called (and it currently is, AFAICT), on correctly inited val and flags.

I know that bitfield order is implementation defined, but I was sure that all the bifields of a same type were equal.
In other words, `memset()/memcpy()` works, this has to work as well.